apps/api/src/modules/payments/payments.service.ts

```diff
@@ -541,25 +544,46 @@ export default class PaymentsService {
     // Attempt to find card (cardId could be source or db ID)
     const card = await PaymentCardModel.query(trx).findById(cardId)
 
-    // Create payment using Circle API
+    // Circle only accepts loopback addresses
+    const verificationHostname = Configuration.webUrl.includes('localhost')
+      ? 'http://127.0.0.1:3000'
+      : Configuration.webUrl
```

Comment:
Let me know if you have any ideas how to setup better. Loopback addresses can be used for the Circle verification URLs, but localhost cannot.